docs/resources/metric_condition.md

```diff
@@ -144,7 +144,7 @@ Optional:
 - `spans` (Block List, Max: 1) (see [below for nested schema](#nestedblock--metric_query--spans))
 - `timeseries_operator` (String)
 - `timeseries_operator_input_window_ms` (Number) Unit specified in milliseconds, but must be at least 30,000 and a round number of seconds (i.e. evenly divisible by 1,000)
-- `tql` (String, Deprecated) Deprecated, use query_string instead
```

Comment:
this was confusing; `metric_condition` never had a `query_string` field but this was shared with `metric_dashboard` (which also lacks a `query_string` field).

updated to say the correct thing, namely -- migrate to the new resource and then name the field correctly